### PR TITLE
upgrade @oasisprotocol/ledger and related

### PR DESCRIPTION
**Pull request by pro-wh**
so we can support version 2.x of the Oasis Ledger app

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -10267,6 +10331,25 @@ prop-types@15.7.2, prop-types@^15.6.2, prop-types@^15.7.2:
...
     react-is "^16.8.1"
 
+protobufjs@~6.11.2:
+  version "6.11.2"
+  resolved "https://registry.yarnpkg.com/protobufjs/-/protobufjs-6.11.2.tgz#de39fabd4ed32beaa08e9bb1e30d08544c1edf8b"
+  integrity sha512-4BQJoPooKJl2G9j3XftkIXjoC9C0Av2NOrWmbLWT1vH32GcSUHjM0Arra6UfTsVyfMAuFzaLucXn1sadxJydAw==
+  dependencies:
+    "@protobufjs/aspromise" "^1.1.2"
+    "@protobufjs/base64" "^1.1.2"
+    "@protobufjs/codegen" "^2.0.4"
```
Comment (pro-wh) [inline yarn.lock:10341]: are you telling me a package called "codegen" is a runtime dependency :skull:

File: yarn.lock [hunk 2]
```diff
@@ -1777,54 +1777,44 @@
...
-  resolved "https://registry.yarnpkg.com/@ledgerhq/hw-transport-webusb/-/hw-transport-webusb-5.46.0.tgz#b8082ef1767b88ebd45f06b235d166736e0f360d"
-  integrity sha512-X5Meh+78nYqzOgOMnqlYQslAzyPOmlgcc+DDtSoWOOwCJ+sCHQmmi+18/aoVgoLtVRpqGjVGxmCBYYX+f3EkOA==
-  dependencies:
-    "@ledgerhq/devices" "^5.46.0"
-    "@ledgerhq/errors" "^5.46.0"
-    "@ledgerhq/hw-transport" "^5.46.0"
-    "@ledgerhq/logs" "^5.46.0"
-
-"@ledgerhq/hw-transport-webusb@^5.41.0":
-  version "5.53.1"
```
Comment (pro-wh) [inline yarn.lock:1806]: we had two versions of this library before this :skull:

File: yarn.lock [hunk 3]
```diff
@@ -1875,31 +1865,32 @@
...
     js-sha512 "^0.8.0"
+    protobufjs "~6.11.2"
     tweetnacl "^1.0.3"
 
-"@oasisprotocol/client@^0.1.0-alpha3":
-  version "0.1.0-alpha4"
-  resolved "https://registry.yarnpkg.com/@oasisprotocol/client/-/client-0.1.0-alpha4.tgz#d5ce6ed2cdf6c9c9a2994634727133ca173c2538"
-  integrity sha512-iXHGbJMwIWGuCDLABQlqkauCnGk6gtaMiurS8+wlhX3IA9QbW4ZL4/tX3mhFCYvSZY/eVHE9FGSF17rhZt0zpQ==
+"@oasisprotocol/client@^0.1.0-alpha5":
+  version "0.1.0-alpha5"
```
Comment (pro-wh) [inline yarn.lock:1891]: why won't yarn use 0.1.0-alpha7 for this

**Comment by pro-wh** [comment]
cc #148

File: yarn.lock (hunk 3 above)
Comment (lukaw3d) [reply yarn.lock:1891]: I think reinstalling @oasisprotocol/client-ext-utils@0.1.0-alpha3 would fix it

Comment (pro-wh) [reply yarn.lock:1891]: woa it does, spooky!

thanks

**Comment by pro-wh** [comment]
note that this puts us over the edge where common.js exists, and the extension won't work in production build

**Comment by pro-wh** [comment]
with the upgrade to client 0.1.0-alpha7+, I'm adding a thing to remove the eval ): ): ):

**Comment by lukaw3d** [comment]
Nice! Still has an issue with building, right?

**Comment by pro-wh** [comment]
yup, should still have that issue!

**Comment by pro-wh** [comment]
Let's revisit this after #156

**Comment by lvshaoping007** [comment]
hi , does this version support paratime ?

**Comment by pro-wh** [comment]
I'm mostly aware that it adds support for on-chain governance voting. I haven't heard about whether it adds paratime support or not.

**Comment by pro-wh** [comment]
great, we've merged #156, and I've freshened this up.

**Comment by pro-wh** [comment]
rebased again. now builds

**Review by lukaw3d — APPROVED** [review]

**Pull request merged**